liquibase-core/src/main/java/liquibase/diff/output/StandardObjectChangeFilter.java

```diff
@@ -36,8 +36,12 @@ public class StandardObjectChangeFilter implements ObjectChangeFilter {
     public StandardObjectChangeFilter(FilterType type, String filter) {
         this.filterType = type;
         if (databaseObjects.isEmpty()) {
```

Comment:
@ettavolt is the first `IF` required because we have the same comparison inside of the `sinchronized` block?

Reply:
Rereading Wiki, I thought there could be a problem with nothing being `volatile`, potentially giving different results for the second conditional (which is inside `synchronized`). How, java.util.SynchornizedCollections don't use any `volatile` to provide thread-safety guarantee.

That said, current code relies on the fact that there is just one instance of writing, which might intendedly change in the future. To be on a safe side, let me initialize the field to null, and then set it to an unmodifiable list.